### ISSUE TITLE
Column Headers do not Wrap as Expected

### ISSUE THREAD

**Issue by rakuhn** [NathanaelA/fluentreports#43]
```
var pageHeader = function(r) {
    r.fontSize(9);
    r.band([
      {data: 'Favorite Ice Cream', width: 70, align: 'left'}, 
      {data: 'Least Favorite Ice Cream', width: 70, align: 'right'} 
      ], {fontBold: true, wrap: true});
  }
  var detail = function(r, row) {
    r.fontsize(9);
    r.band([
      {data: row.favorite, width: 70, align: 'left'}, 
      {data: row.least, width: 70, align: 'right'}
    ]);
  }

  var report = new Report("myReport.pdf")
    .data([{favorite: 'Butter Pecan', least: 'Pistachio'}])
    .pageHeader(pageHeader)
    .detail(detail);

  Report.render(function(err, name) { console.log("The report was saved to", name); });
```
In the code in the example above, the least favorite ice cream header is left aligned because it is wrapping.  I would like the wrapped text to be right aligned.

**Comment by ahenager** [comment]
KellPro is interested in sponsoring the fix for this item.

**Comment by NathanaelA** [comment]
@rakuhn - Thanks for the sample; I can confirm that it isn't wrapping properly.

**Comment by NathanaelA** [comment]
This has been fixed in v1.08, thanks to Kellpro for Sponsoring the fix!
